src/pages/apis/drug/drugsfda/_meta.yaml

```diff
@@ -0,0 +1,12 @@
+documentTitle: openFDA › Drugs › Drugs@FDA
+label: Drugs@FDA
+title: Drugs@FDA
+description: "Drug products are identified and reported using a unique, three-segment number, called the National Drug Code (NDC), which serves as a universal product identifier for drugs."
+datasets:
+  - drugsfda
+path: /apis/drug/drugsfda
+api_path: /drug/drugsfda
+start: 20090101
```

Comment:
@beardedfinch The oldest drug submission is 19390209; we should probably adjust the `start` value, too.

Reply:
Fixed.